lib/transport_web/templates/user/organizations.html.eex

```diff
@@ -12,7 +12,7 @@
           <%= organization["name"] %>
         </div>
         <div>
-            <a href="mailto:contact@transport.data.gouv.fr" class="button" role="link">
+            <a href=<%= user_path @conn, :org_datasets, organization["slug"] %> class="button" role="link">
```

Comment:
```elixir
   <%= user_path(@conn, :organization_datasets, organization.slug %>
```

?

That involves:
- use parenthesis
- change route name
- use a Struct

Reply:
I don't want to use struct now.